rfcs/interop_2022.md

```diff
@@ -0,0 +1,133 @@
+
+- [`interop-2021-aspect-ratio`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-aspect-ratio)
+- [`interop-2021-flexbox`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-flexbox)
+- [`interop-2021-grid`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-grid)
+- [`interop-2021-position-sticky`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-position-sticky)
+- [`interop-2021-transforms`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-transforms)
+
+[Query for `interop-2021-*`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-aspect-ratio%20or%20label%3Ainterop-2021-flexbox%20or%20label%3Ainterop-2021-grid%20or%20label%3Ainterop-2021-position-sticky%20or%20label%3Ainterop-2021-transforms)
+
+### Metrics
+
+Scores (between 0% and 100%) will be computed from test suite pass rates for each focus area and the effort as a whole. In total there are 15 focus areas, of which 5 are carried forward from Interop 2021. All areas are given equal weight, meaning that the new areas make up 2/3 of the score and the old ones 1/3 of the score.
+
+Scoring in detail:
+
+- Every test is scored between 0 and 1. For tests with subtests, the score is the number of passes divided by the number of subtests. This avoids tests with many subtests being given much larger weight than for example reftests.
+- Every area is scored between 0 and 1, adding up the scores and dividing by the number of tests.
+- The overall score is the sum of area scores divided by the number of areas (15).
+
+Scores are presented as percentages.
+
+TODO: decide on floating point vs. integer vs. rational numbers
+
+### Dashboard
+
+A public dashboard tracking the metrics over time will be available at https://wpt.fyi/interop-2022. The dashboard will include at least:
+
+- An explanation of the scope, which is the 15 focus areas, not the whole web platform.
+- A graph of the metric over time for at least Chrome, Firefox and Safari, which can show both the overall Interop 2022 metric, the Interop 2021 metric which makes up 1/3 of the score, and each of the 15 focus areas individually.
+- A brief description of each focus area, with links to specs, tests on wpt.fyi, and documentation on MDN.
+
+Additional features might be added, see https://github.com/web-platform-tests/interop-2022/issues/45 for discussion.
+
+### How to nominate an area
```

Comment:
This feels pretty weird. I guess we should have had an RFC for the process and an RFC for the outcome. But I think this section should be rewritten in the past tense: "The process for nominating areas was…", since in theory one might judge this RFC by whether one thought a reasonable process had been used.

Reply:
I didn't read this comment until now, but I deleted this bit yesterday.